### ISSUE TITLE
Display changeset always creates a new layer

### ISSUE THREAD

**Issue by maning** [JOSM/changeset-map#3]
The **Display changeset** button always creates a new layer, can we instead show or activate the previous layer of the same changeset?  
![screen shot 2018-02-09 at 09 44 41](https://user-images.githubusercontent.com/353700/36033076-e888a042-0d7d-11e8-9f24-3c05be7db65d.png)

**Comment by Rub21** [comment]
@maning  yep, I think we should avoid display the same changeset !! but we could display different changesets at the same time!!  also, i am thinking maybe it should be an option to display one or bunch of changesets
